Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/actions/FilterActionOnCompositeFilterCriteria.java

```diff
@@ -45,7 +45,8 @@ public boolean execute() {
             analyst.filter(postActionFilterCriteria);
             return true;
         } catch (Exception e) {
-            Messagebox.show("Error in filtering. Error message: " + e.getMessage());
+            // LOGGER.error("Error in filtering. Error message: " + e.getMessage());

```

Comment:
Can this be removed, or better, uncommented? If uncommented, include `e` in the logging call.